### ISSUE TITLE
Version 2 doesn't run on Mint

### ISSUE THREAD

**Issue by eugenialoli** [mfat/systemd-pilot#4]
Appimage with v2.0:
*** stack smashing detected ***: terminated
Aborted (core dumped)

also, consider take it a bit slower with the versioning, jumping from 1.2 to 2 . :-)

**Comment by mfat** [comment]
Thanks @eugenialoli . The reason for the jump to 2.0 is that a few key features were added :)
The AppImage runs fine on ubuntu 24.10. Wondering what could be the problem with Mint.

**Comment by eugenialoli** [comment]
Mint 22 is based on 24.04, which is the ubuntu LTS, not 24.10.

**Comment by mfat** [comment]
Can you give this a try? https://github.com/mfat/systemd-pilot/releases/download/v2.0/systemd-pilot-2.0-x86_64.AppImage

**Comment by eugenialoli** [comment]
Yes, this one works, but every time I click on each of the tabs, it takes 2-3 seconds, it gets "stuck". Also, the icon is low res compared to the original 2.0 appimage.

**Comment by mfat** [comment]
Is your system's gtk theme applied correctly?

And did you try the flatpak or deb package too?

**Comment by eugenialoli** [comment]
I usually prefer appimages, so I only tried that. GTK is applied fine. I'm on a Macbook Air 2015, which has 2400 cpu passmark points. If your computer is very fast (e.g. over 15,000 points, check your cpu against the passmark benchmark) it's possible that you're not seeing what I'm seeing.

**Comment by mfat** [comment]
I'll try to rebuild the appimage on an older LTS release to ensure compatibility.
